lldb/include/lldb/Core/ModuleList.h

```diff
@@ -33,6 +33,14 @@
 #include <cstddef>
 #include <cstdint>
 
+namespace llvm {
```

Comment:
I guess this can't be upstream?

Do you want to add `// START CAS` so conflicts are easier to manage?